src/functionalTest/resources/scenarios/RIA-3720-case-listed-notification-remote-hearing.json

```diff
@@ -0,0 +1,93 @@
+{
+  "description": "RIA-3720 Send case listed notification to case officer, legal rep and home office in a remote hearing",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "AdminOfficer",
+    "input": {
+      "id": 3720,
+      "eventId": "listCase",
+      "state": "prepareForHearing",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "appellantInUk": "No",
+          "remoteVideoCallTribunalResponse": "Test",
+          "ariaListingReference": "LP/12345/2021",
+          "hearingCentre": "birmingham",
+          "listCaseHearingCentre": "remoteHearing",
+          "listCaseHearingDate": "2021-02-19T14:25:15.000",
+          "legalRepresentativeEmailAddress": "{$TEST_LAW_FIRM_A_USERNAME}"
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "minimal-appeal-submitted.json",
+      "replacements": {
+        "appellantInUk": "No",
+        "remoteVideoCallTribunalResponse": "Test",
+        "hearingCentre": "birmingham",
+        "listCaseHearingCentre": "remoteHearing",
+        "listCaseHearingDate": "2021-02-19T14:25:15.000",
+        "legalRepresentativeEmailAddress": "{$TEST_LAW_FIRM_A_USERNAME}",
+        "notificationsSent": [
+          {
+            "id": "3720_CASE_LISTED_CASE_OFFICER",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          },
+          {
+            "id": "3720_CASE_LISTED_LEGAL_REPRESENTATIVE",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          },
+          {
+            "id": "3720_CASE_LISTED_HOME_OFFICE",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          }
+        ]
+      }
+    },
+    "notifications": [
+      {
+        "reference": "3720_CASE_LISTED_CASE_OFFICER",
+        "recipient": "{$hearingCentreEmailAddresses.birmingham}",
+        "subject": "Immigration and Asylum appeal: case listed",
+        "body": [
+          "PA/12345/2019",
+          "LP/12345/2021"
+        ]
+      },
+      {
+        "reference": "3720_CASE_LISTED_LEGAL_REPRESENTATIVE",
+        "recipient": "{$TEST_LAW_FIRM_A_USERNAME}",
+        "subject": "Immigration and Asylum appeal: check the details of your hearing",
+        "body": [
+          "PA/12345/2019",
+          "LP/12345/2021",
+          "CASE001",
+          "Talha Awan",
+          "{$iaExUiFrontendUrl}",
+          "19 Feb 2021",
+          "14:25"
+        ]
```

Comment:
Your functional test should check for all the placeholder fields in the template.

- hearingCentreAddress (needs to be hearingCentreName)
- remoteVideoCallTribunalResponse
- hearingRequirementVulnerabilities
- hearingRequirementMultimedia
- hearingRequirementSingleSexCourt
- hearingRequirementInCameraCourt
- hearingRequirementOther
- customerServicesTelephone
- customerServicesEmail